tests/components/august/test_init.py

```diff
@@ -44,14 +50,41 @@ def test_lock_throws_august_api_http_error():
     data.set_mocked_locks([lock])
     last_err = None
     try:
-        data.unlock("mockdeviceid1")
+        data.unlock("mocklockid1")
     except HomeAssistantError as err:
         last_err = err
     assert (
-        str(last_err) == "Mocked Lock 1: This should bubble up as its user consumable"
+        str(last_err)
+        == "mocklockid1 Name: This should bubble up as its user consumable"
     )
 
 
+class TestAugustData(unittest.TestCase):
```

Comment:
New tests must be (async) standalone pytest test functions.

Reply:
@MartinHjelmare Is there a spec for how the tests should be written?  I feel like I'm shooting in the dark here.  Sorry if I missed it.

Reply:
Use the example I Iinked in the earlier PR as a template.